plugins/services/src/js/validators/VipLabelsValidators.js

```diff
@@ -0,0 +1,62 @@
+import {isEmpty} from '../../../../../src/js/utils/ValidatorUtil';
+import ServiceConfigUtil from '../utils/ServiceConfigUtil';
+import NetworkValidatorUtil from '../../../../../src/js/utils/NetworkValidatorUtil';
+import {findNestedPropertyInObject} from '../../../../../src/js/utils/Util';
+import {VIP_LABEL_VALUE_REGEX} from '../../../../../src/js/constants/Networking';
+
+function checkServiceEndpoints(ports, pathPrefix) {
+  return ports.reduce(function (memo, port, index) {
+    const labels = port.labels || {};
+    const vipLabels = Object.keys(labels)
+      .filter(ServiceConfigUtil.matchVIPLabel);
+
+    vipLabels.forEach(function (label) {
+      const [_address, vipPort] = labels[label].split(':');
```

Comment:
Please use `RegExp.prototype.exec` to split the label into name/ip and port to properly validate values.